lib/importers/tp/saver.rb

```diff
@@ -67,8 +72,17 @@ class IncludedCallOutcome
           'REFER TO CIT-A ENGLAND AND WALES'
         ].freeze
 
-        def satisfied?(call)
-          INVALID_OUTCOMES.exclude?(call.outcome.upcase)
+        INVALID_CALL_OUTCOMES = [
+          'TEST CODE'
+        ].freeze
+
+        def satisfied?(filter_type, call)
+          case filter_type
+          when :where_did_you_hear
```

Comment:
Actually unless I'm missing something the case statement is pointless here. You're doing the same thing regardless so surely this should just be inlined?